src/setup-beam.js

```diff
@@ -133,15 +133,15 @@ async function getElixirVersion(exSpec0, otpVersion) {
       `Requested Elixir version (from spec ${exSpec0}) not found in build listing`,
     )
   }
-  const otpMatch = otpVersion.match(/^(?:OTP-)?([^\.]+)/)
+  const otpMatch = otpVersion.match(/^(?:OTP-)?([^.]+)/)
```

Comment:
Not a real problem, but not visible with the previously linting procedure, either.